src/js/entities-service/entities/clinicians.js

```diff
@@ -70,13 +69,17 @@ const _Model = BaseModel.extend({
     return !this.get('last_active_at');
   },
   isActive() {
-    const hasTeam = !!this.get('_team');
+    const hasTeam = this.hasTeam();
     const hasGroups = !!size(this.get('_groups'));
     const lastActive = this.get('last_active_at');
 
     return hasTeam && hasGroups && lastActive;
   },
+  hasTeam() {
+    const team = this.get('_team');
 
+    return team && team !== NIL_UUID;
```

Comment:
Could just be `team !== NIL_UUID`? Since a clinician will always have a team assigned, even if it's just the `NIL_UUID`.

But this covers any potential scenario where that doesn't happen and `team` is `null` for a clinician.

Reply:
I think it should be kept like this, in case we were to switch to `id: null` again.

Reply:
Typically I would not want to support both cases explicitly as we'd be testing for both branches and we'd have no indication of dead code.  However in this case I suspect removing the `team &&` will be more headaches as I suspect we're not setting it across all of the fixtures, but removing the `team &&` would probably surface where those are..

I think it we want to handle both `null` and `NIL_UUID` in most cases we should think through a generic `hasRelationship` solution for the base model, which probably isn't a bad idea anyhow as we have a number of places where we have unique checks for relationships in the app.  The trick will be differentiating between 1-1 and 1-many.  

Anyhow.. I think this is ok for this bugfix PR.  Can we move this method up by the `getTeam` though?

Reply:
That makes sense.

Ya, I originally had it as just `return team !== NIL_UUID;` but that caused a test to fail since team is set to `null` in the tests. So it was a decision between adding the `nil_uuid` in the tests or doing `return team && team !== NIL_UUID;` in the app.

I moved the `hasTeam()` method up below the `getTeam()` method in that last commit I just pushed.